op-guide/recommendation.md

```diff
@@ -1,51 +1,74 @@
 ---
-title: 部署建议
+title: 软件和硬件环境要求
 category: deployment
 ---
 
-# 部署建议
+# TiDB 软件和硬件环境要求
 
-阅读本章前，请先阅读 [TiDB 整体架构](../overview.md#tidb-整体架构)。
+## 概述
 
-## TiDB 集群各个组件的硬件消耗情况及推荐配置
+TiDB 作为一款开源分布式 NewSQL 数据库，可以很好的部署和运行在 Intel 架构服务器环境及主流虚拟化环境，并支持绝大多数的主流硬件网络。作为一款高性能数据库系统，支持主流的 Linux 操作系统环境。
 
-| 组件 | 消耗硬件资源 | 推荐硬件配置 |
-| ---- | -------- | -------- |
-| TiDB | CPU、内存 |32+ 核/64G+ 内存 |
-| TiKV | CPU、内存、磁盘 IO |32+ 核/64G+ 内存/200G+ SSD硬盘|
-| PD   | CPU、内存、磁盘 IO |8+ 核/16G+ 内存/200G+ SSD硬盘 |
+## Linux 操作系统版本要求
 
-备注：
+| Linux 操作系统平台       | 版本         |
+| :----------------------- | :----------: |
+| Red Hat Enterprise Linux | 7.3 及以上   |
+| CentOS                   | 7.3 及以上   |
+| Oracle Enterprise Linux  | 7.3 及以上   |
+| Ubuntu LTS               | 14.04 及以上 |
 
-* OLTP 线上业务强烈建议使用 SSD 硬盘，否则在一些场景下会有性能问题。
-* TiKV 硬盘大小建议不要超过500G（防止硬盘损坏时，数据恢复耗时过长）
+> 注：Oracle Enterprise Linux 只支持 Red Hat 兼容内核 (RHCK) , 不支持 Oracle Enterprise Linux 提供的 Unbreakable Enterprise Kernel。
 
-## 整体部署方案
+> 注：以上所支持的 Linux 操作系统包括部署和运行在物理服务器以及 VMWare, KVM, XEN 主流虚拟化环境。
```

Comment:
ref: https://www.vmware.com/help/trademarks.html

Its trademark is `VMware`, not `VMWare`